sdk/mysql/Azure.ResourceManager.MySql/api/Azure.ResourceManager.MySql.netstandard2.0.cs

```diff
@@ -254,1069 +259,1027 @@ public partial class QueryStatisticData : Azure.ResourceManager.Models.ResourceD
+        public virtual System.Threading.Tasks.Task<Azure.ResourceManager.ArmOperation<Azure.ResourceManager.MySql.MySqlVirtualNetworkRuleResource>> UpdateAsync(Azure.WaitUntil waitUntil, Azure.ResourceManager.MySql.MySqlVirtualNetworkRuleData data, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
-    public partial class WaitStatisticCollection : Azure.ResourceManager.ArmCollection
+    public partial class MySqlWaitStatisticCollection : Azure.ResourceManager.ArmCollection
     {
-        protected WaitStatisticCollection() { }
+        protected MySqlWaitStatisticCollection() { }
         public virtual Azure.Response<bool> Exists(string waitStatisticsId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<bool>> ExistsAsync(string waitStatisticsId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.Response<Azure.ResourceManager.MySql.WaitStatisticResource> Get(string waitStatisticsId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.Pageable<Azure.ResourceManager.MySql.WaitStatisticResource> GetAll(Azure.ResourceManager.MySql.Models.WaitStatisticsInput input, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.AsyncPageable<Azure.ResourceManager.MySql.WaitStatisticResource> GetAllAsync(Azure.ResourceManager.MySql.Models.WaitStatisticsInput input, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.MySql.WaitStatisticResource>> GetAsync(string waitStatisticsId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.MySql.MySqlWaitStatisticResource> Get(string waitStatisticsId, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.MySql.MySqlWaitStatisticResource> GetAll(Azure.ResourceManager.MySql.Models.MySqlWaitStatisticsInput input, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.MySql.MySqlWaitStatisticResource> GetAllAsync(Azure.ResourceManager.MySql.Models.MySqlWaitStatisticsInput input, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Similar question on the GetAll method

Reply:
It's probably because those statistics/query result without filters is too big and therefore, some filter parameters are required. Need further confirmation with service team.